komodo/ask_for_tags.py

```diff
@@ -0,0 +1,218 @@
+from github.Repository import Repository
+
+not_in_komodo_releases_project_msg = (
+    "We expect this script to be run in the komodo-releases project"
+)
+
+
+def get_packages_that_use_main() -> List[str]:
+    bleeding_release_file = "releases/matrices/bleeding.yml"
+    duplicate_pkgs = ["everest-models"]  # this is included in the everest repo
+    try:
+        with open(bleeding_release_file, "r", encoding="utf-8") as bleeding_raw:
+            bleeding = yaml.safe_load(bleeding_raw)
+    except FileNotFoundError as err:
+        raise FileNotFoundError(not_in_komodo_releases_project_msg) from err
+
+    main_pkgs_in_bleeding = [
+        pkg.lower()
+        for pkg, version in bleeding.items()
+        if version == "main" and pkg not in duplicate_pkgs
+    ]
+    return main_pkgs_in_bleeding
+
+
+def get_repos(gh_client: Github, packages: List[str]) -> List[Repository]:
+    """tries to download repositories based on provided package names, using some
+    predefined fork names. throws if repo for a package could not be found / accessed
+    """
+
+    DEFAULT_FORK = "equinor"
+    ALTERNATE_FORK = "tno-everest"
+
+    repos = []
+    for package in packages:
+        try:
+            repo = gh_client.get_repo(f"{DEFAULT_FORK}/{package}")
+        except UnknownObjectException:
+            repo = gh_client.get_repo(f"{ALTERNATE_FORK}/{package}")
+        repos.append(repo)
+
+    packages_without_repo = [
+        pkg for pkg in packages if pkg not in [repo.name.lower() for repo in repos]
+    ]
+    if packages_without_repo:
+        errMsg = (
+            "Could not find / access repo for following packages: "
+            f"{packages_without_repo}"
+        )
+        raise RuntimeError(errMsg)
+    return repos
+
+
+def get_latest_commit(repo: Repository) -> Commit:
+    DEFAULT_BRANCH = "main"
+    ALTERNATE_BRANCH = "master"
+    try:
+        return repo.get_branch(DEFAULT_BRANCH).commit
+    except GithubException:
+        return repo.get_branch(ALTERNATE_BRANCH).commit
+
+
+def get_commit_of_latest_release(repo: Repository) -> Commit:
+    last_release = repo.get_latest_release()
+    tags = repo.get_tags()
+    release_tag = [tag for tag in tags if tag.name == last_release.tag_name][0]
+    return release_tag.commit
+
+
+def find_repos_with_changes_since_last_release(
+    repos: List[Repository],
+) -> List[Repository]:
+    repos_with_changes = []
+    for repo in repos:
+        last_commit = get_latest_commit(repo)
+        release_commit = get_commit_of_latest_release(repo)
+        if last_commit != release_commit:
+            repos_with_changes.append(repo)
+    return repos_with_changes
+
+
+def get_scout_maintainers(gh_client: Github) -> Mapping[str, str]:
+    scout_repo = gh_client.get_repo("equinor/scout")
+    projects_file = scout_repo.get_contents("projects.md")
+    projects_file_contents = base64.b64decode(projects_file.content).decode()
```

Comment:
More Equinor specific stuff. Maybe we can generalize this, I'm not sure. We've been wondering if other parts of the code could benefit from a local config file -- possibly this is `locations.yaml` or maybe some new thing. This seems like the kind of thing that could go in there.